doc/classes/Transform3D.xml

```diff
@@ -62,7 +62,7 @@
 		<method name="affine_inverse" qualifiers="const">
 			<return type="Transform3D" />
 			<description>
-				Returns the inverted version of this transform. Unlike [method inverse], this method works with almost any [member basis], including non-uniform ones, but is slower. See also [method Basis.inverse].
+				Returns the inverted version of this transform. Unlike [method inverse], this method works with almost any [member basis], including non-uniform ones, but is slower. See also [method inverse].
```

Comment:
I don't think this needs to be changed.

It seems like this intentionally links to `Basis.inverse`. `Transform3D.inverse` also links to `Basis.inverse`:
https://github.com/godotengine/godot/blob/296a30d73f428105dc228cb0531dea294bf43b04/doc/classes/Transform3D.xml#L78-L84

If we don't want to link to `Basis.inverse` for whatever reason, then the whole "see also" sentence should be removed, since the description already links to `Transform3D.inverse` in the second sentence.

Reply:
Right. There's a "see also" to be removed in Transform2D's equivalent, though.